config/src/logger.rs

```diff
@@ -29,77 +22,49 @@ pub fn into_tracing_level(level: Level) -> tracing::Level {
     }
 }
 
-/// Wrapper for [`Handle`] to implement [`ReloadMut`]
-#[derive(From)]
-pub struct ReloadHandle<T>(pub Handle<LevelFilter, T>);
-
-impl<T: Subscriber + Debug> ReloadMut<Level> for ReloadHandle<T> {
-    fn reload(&mut self, level: Level) -> Result<(), ReloadError> {
-        let level_filter =
-            tracing_subscriber::filter::LevelFilter::from_level(into_tracing_level(level));
-
-        Handle::reload(&self.0, level_filter).map_err(|err| {
-            if err.is_dropped() {
-                ReloadError::Dropped
-            } else {
-                ReloadError::Poisoned
-            }
-        })
-    }
-}
-
-/// Wrapper around [`Level`] for runtime upgrades.
-#[derive(Debug, Clone, Default, Deref, DerefMut, Deserialize, Serialize)]
-#[repr(transparent)]
-#[serde(transparent)]
-pub struct SyncLevel(handle::SyncValue<Level, handle::Singleton<Level>>);
-
-impl From<Level> for SyncLevel {
-    fn from(level: Level) -> Self {
-        Self(level.into())
-    }
-}
-
-impl PartialEq for SyncLevel {
-    fn eq(&self, other: &Self) -> bool {
-        self.0.value() == other.0.value()
-    }
-}
-
-impl Eq for SyncLevel {}
-
 /// 'Logger' configuration.
 #[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize, Proxy, Documented)]
 #[serde(rename_all = "UPPERCASE")]
+#[config(env_prefix = "LOG_")]
+// `tokio_console_addr` is not `Copy`, but warning appears without `tokio-console` feature
```

Comment:
Now that it has a type of `SocketAddr`, it _is_ actually `Copy`, so it can be derived here

Reply:
It is actually not:

```
error[E0204]: the trait `std::marker::Copy` cannot be implemented for this type
  --> config/src/logger.rs:26:17
   |
26 | #[derive(Debug, Copy, Clone, PartialEq, Eq, Deserialize, Serialize, Proxy, Documented)]
   |                 ^^^^
...
37 |     pub tokio_console_addr: SocketAddr,
   |     ---------------------------------- this field does not implement `std::marker::Copy`
   |
```

And it cannot be `Copy` because `SocketAddr::Host` -> `SocketAddrHost` contains `BoxedString` within.

Reply:
Oh, I confused it with std's `SocketAddr`